CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+# Build
+The declaration files are auto generated(as there are new corvid APIs on a monthly base) 
+
+To generate the files, simply run
+`npm i && npm run build`
+
+# Test
+`npm run test`
+
+# Publish to npm
+On master branch, run [`npm version [<newversion> | major | minor | patch | ...] -m <commit message>`](https://docs.npmjs.com/cli/version)
```

Comment:
Why is it different here from corvid-cli in which we don't do it directly on master? 🤔

Reply:
Because it works better :)